### PR TITLE
Core: Add highlight.js to es6 transpiled modules

### PR DESCRIPTION
**Pull request by tooppaaa**
Issue: IE 11, #12399 #12179

## What I did

Transpile highlight.js

## How to test

- Be bold and launch IE 11

### CODE REVIEW

**Comment by tooppaaa** [comment]
@gaetanmaisse any idea on fixing yarn 2 while keeping the highlight.js transpilation ?

File: lib/core/src/server/common/es6Transpiler.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { plugins } from './babel';
 const es6Transpiler = () => {
   // TODO: generate regexp using are-you-es5
 
-  const include = /[\\/]node_modules[\\/](@storybook\/node-logger|are-you-es5|better-opn|boxen|chalk|commander|find-cache-dir|find-up|fs-extra|json5|node-fetch|pkg-dir|resolve-from|semver)/;
+  const include = /[\\/]node_modules[\\/](@storybook\/node-logger|are-you-es5|better-opn|boxen|chalk|commander|find-cache-dir|find-up|fs-extra|highlight.js|json5|node-fetch|pkg-dir|resolve-from|semver)/;
```
Comment (shilman) [inline lib/core/src/server/common/es6Transpiler.js:6]: `highlight\.js`?

**Review by shilman — CHANGES_REQUESTED** [review]
Small tweak?

File: lib/core/src/server/common/es6Transpiler.js (hunk 1 above)
Comment (tooppaaa) [reply lib/core/src/server/common/es6Transpiler.js]: Done but still not working in yarn 2 :(

**Comment by gaetanmaisse** [comment]
@tooppaaa I discussed a bit about that with @yannbf and we have some options in mind. Let's talk in the next few days.

**Comment by yannbf** [comment]
LGTM! I'm not sure if there are possible side effects to this but seems solid. Of course there has to be some documentation as well, maybe on FAQ?

**Comment by tooppaaa** [comment]
Just waiting to be sure we want to go to the path of adding options like this in main.js @shilman @tmeasday @ndelangen ?

I'm sure @jonniebigodes will be able to point me the right direction for documenting this after :D

**Comment by ndelangen** [comment]
I thought we used prism for code highlighting rather than highlight?

where are we using highlight?

**Comment by jonniebigodes** [comment]
@tooppaaa let me know when you have something created and we can go from there. Sounds good?

**Comment by tooppaaa** [comment]
> where are we using highlight?

Don't know but it's definitely present in the bundles.

**Comment by shilman** [comment]
I think this is no longer needed because of #12948. @tooppaaa WDYT?

**Comment by tooppaaa** [comment]
IE 11 is working ! not needed !

**Pull request closed without merging**